seo/seo.md

```diff
@@ -63,4 +66,9 @@ Review your own portfolio site, connect it to some of the tools learned in this
 
 ### Check for Understanding
 
-Have apprentices present/summarize SEO at work (and potential suggestions for improvement) to a partner. Apprentices can use an example of their choice-- such as a random site from the Internet or their own portfolio site.
+- Have apprentices present/summarize SEO at work (and potential suggestions for improvement) to a partner. Apprentices can use an example of their choice-- such as a random site from the Internet or their own portfolio site.
```

Comment:
Maybe remove line 69... it doesn’t make sense, and your other questions address those things anyway.